EKF/ekf.h

```diff
@@ -465,15 +462,13 @@ class Ekf : public EstimatorInterface
 	bool _is_first_imu_sample{true};
 	uint32_t _baro_counter{0};		///< number of baro samples read during initialisation
 	uint32_t _mag_counter{0};		///< number of magnetometer samples read during initialisation
-	uint64_t _time_last_mag{0};		///< measurement time of last magnetomter sample (uSec)
 	AlphaFilterVector3f _mag_lpf;		///< filtered magnetometer measurement for instant reset(Gauss)
 	AlphaFilterVector3f _accel_lpf;		///< filtered accelerometer measurement for instant reset(Gauss)
 	float _hgt_sensor_offset{0.0f};		///< set as necessary if desired to maintain the same height after a height reset (m)
 	float _baro_hgt_offset{0.0f};		///< baro height reading at the local NED origin (m)
 
 	// Variables used to control activation of post takeoff functionality
 	float _last_on_ground_posD{0.0f};	///< last vertical position when the in_air status was false (m)
-	bool _flt_mag_align_converging{false};	///< true when the in-flight mag field post alignment convergence is being performd
```

Comment:
I wonder if the place where this was used before was accidentally removed? Could you do a quick search through the git history to make sure this wasn't a mistake?

Reply:
You are right! We lost the following assignment during the [mag control refactor](https://github.com/PX4/ecl/pull/662/files#diff-4cac31c333867233bb8a08de1e812a9aL1461).
FYI @bresch @priseborough

Reply:
Thanks for pointing that, it's going to be fixed in https://github.com/PX4/ecl/pull/692